src/components/ProductsDashboard/ProductsDashboard.js

```diff
@@ -1,13 +1,11 @@
 import React, { Component } from 'react';
-import AllProducts from './AllProducts/AllProducts';
 import AddProduct from './AddProduct/AddProduct';
 
 export default class ProductsDashboard extends Component {
   render() {
     return (
       <div>
         <AddProduct />
-        <AllProducts />
```

Comment:
Why AllProducts have been removed? I don't see them being used anywhere else.
Also if this removal was intentional, it's worth checking if the removed components is still used anywhere in the code, not to leave unused components.

Reply:
I removed it because I must spend the time work on this component, to work correctly. What is the best practice to turn off the component, should I add a comment?

Reply:
Depends on the project, but it's not a common situation.
If you want to hide component and not lose the markup, placement, etc. you can just `{false && <AllProducts />}`